docs/management/admin/host-isolation-ov

```diff
@@ -0,0 +1,69 @@
+[[host-isolation-ov]]
+[chapter, role="xpack"]
+= Host isolation
+
+Host isolation allows you to isolate Windows and macOS hosts from your network, blocking communication with any other hosts on your network until the host is released. Isolating a host is useful when responding to malicious activity or to prevent potential attacks, as it prevents lateral movement across other hosts.
```

Comment:
only thing i can think to add is that isolated hosts will still be able to communicate with kibana/ ES